app/version/version.go

```diff
@@ -36,6 +36,10 @@ func GitCommit() (hash string, timestamp string) {
 
 	for _, s := range info.Settings {
 		if s.Key == "vcs.revision" {
+			if len(s.Value) < 7 {
```

Comment:
This isn't critical stuff, rather just use `min(len(x),7]` everywhere: ie. `hash := s.Value[:min(len(v.Value,7)]`